chainerx_cc/chainerx/routines/math.cc

```diff
@@ -770,6 +770,17 @@ Array Cos(const Array& x) {
     return out;
 }
 
+Array Absolute(const Array& x) {
+    Dtype dtype = GetMathResultDtype(x.dtype());
+    Array out = Empty(x.shape(), dtype, x.device());
```

Comment:
You're allocating an array here which is discarded when being assigned the sum of `x_flip_1` and `x_flip_2`.